### ISSUE TITLE
fly launch should create a volume when using an existing config with `[mounts]`

### ISSUE THREAD

**Issue by mrkurt** [superfly/flyctl#729]
When I run `fly launch` in a directory with this config:

```toml
kill_signal = "SIGINT"
kill_timeout = 5
processes = []

[mounts]
  source="myapp_data"
  destination="/home/coder/project"
```

It should create the volume for me.  I believe all it needs to know is the size, at this stage we should already know the region. So if it prompts for size, then runs the equivalent of `fly volumes create myapp_data --size <size>` it'll be pretty magical.

**Comment by rugwirobaker** [comment]
Isn't that the whole point of this block: **https://github.com/superfly/flyctl/blob/02c46ec337f600a46bb7c388a602776c8ab4884a/cmd/launch.go#L355-L380**

**Comment by michaeldwan** [comment]
@rugwirobaker that block creates the volumes requested by the scanner. Kurt is saying that if a fly.toml file has mounts we should detect this and prompt for the size then create as needed.

**Comment by dangra** [comment]
Fixed by #2086
